OpenEdXMobile/res/layout/activity_login.xml

```diff
@@ -2,13 +2,13 @@
 <layout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools">
-
+    
     <android.support.design.widget.CoordinatorLayout
+        android:id="@+id/cl_root"
```

Comment:
need id for title on all screens

Reply:
following story has been created for that
https://openedx.atlassian.net/browse/LEARNER-4408